wasm.md

```diff
@@ -1471,7 +1471,7 @@ A new module instance gets allocated.
 Then, the surrounding `module` tag is discarded, and the definitions are executed, putting them into the module currently being defined.
 
 ```k
-    rule <k> #module(... types: TS, funcs: FS, tables: TABS, mems: MS, globals: GS, elem: EL, data: DAT, start: S,  importDefns: IS, exports: ES,
+    rule <instrs> #module(... types: TS, funcs: FS, tables: TABS, mems: MS, globals: GS, elem: EL, data: DAT, start: S,  importDefns: IS, exports: ES,
                          metadata: #meta(... id: OID, funcIds: FIDS))
           => sequenceDefns(TS)
           ~> sequenceDefns(IS)
```

Comment:
Formatting, line up the right side of `=>` and `~>` and `...` with the right side of `<instrs>`.

Reply:
Done